tabbycat/draw/utils.py

```diff
@@ -22,3 +31,44 @@ def annotate_npullups(teams, until):
 
     for team in teams:
         team.npullups = npullups_by_team_id.get(team.id, 0)
+
+
+def send_mail_to_adjs(round):
+    tournament = round.tournament
+    draw = round.debate_set.all().prefetch_related('debateadjudicator_set')
+
+    subject = Template(tournament.pref('adj_email_subject_line'))
+    body = Template(tournament.pref('adj_email_message'))
+    messages = []
+
+    def _assemble_panel(adjs):
+        adj_string = []
+        for adj in adjs:
+            adj_string.append("%s (%s)" % (adj.adjudicator.name, adj.get_type_display()))
+
+        return ", ".join(adj_string)
+
+    for debate in draw:
+        adjs = debate.debateadjudicator_set.all()
```

Comment:
The method `debate.adjudicators.with_positions()` iterates through the list in order of position (chairs, their panellists, then trainees). It yields tuples `(adj, adjtype)` where `adj` is an `Adjudicator` (not a `DebateAdjudicator`) and `adjtype` is one of the `AdjudicatorAllocation.POSITION_*` values (which are not the same as the `DebateAdjudicator_*` values, because the former uses a different value for chairs who don't have panellists).

`debate.adjudicators` is an `AdjudicatorAllocation` object, which is a class whose purpose is to provide a nicer interface than the `DebateAdjudicator` [related manager](https://docs.djangoproject.com/en/2.0/ref/models/relations/) that `debateadjudicator_set` returns.